doc/cug.tex

```diff
@@ -5865,36 +5958,34 @@ \subsection{Suite And Task Event Handling}
 the environment you must use [cylc] $\rightarrow$ [[environment]],
 not task runtime environments.
 
-Here is an example suite that tests the {\em retry} and {\em failed} events.
-The handler in this case simply echoes its command line arguments to
-suite stdout.
+The following 2 \lstinline=suite.rc= snippets are examples on how to specify
+event handlers using the alternate methods:
 
 \lstset{language=suiterc}
 \begin{lstlisting}
-[scheduling]
-    initial cycle point = 20100808T00
-    final cycle point = 20100810T00
-    [[dependencies]]
-        [[[T00]]]
-            graph = "foo => bar"
 [runtime]
     [[foo]]
         retry delays = PT0S, PT30S
-        script = """
-echo TRY NUMBER: $CYLC_TASK_TRY_NUMBER
-sleep 10
-# retry twice and succeed on the final try,
-# but fail definitively in the final cycle.
-if (( CYLC_TASK_TRY_NUMBER <= 2 )) || \
-    (( CYLC_TASK_CYCLE_POINT == CYLC_SUITE_FINAL_CYCLE_POINT )); then
-    echo ABORTING
-    /bin/false
-fi"""
+        script = test "${CYLC_TASK_TRY_NUMBER}" -eq 2
         [[[event hooks]]]
-            retry handler = "echo !!!!!EVENT!!!!! "
-            failed handler = "echo !!!!!EVENT!!!!! "
+            retry handler = "echo '!!!!!EVENT!!!!!' "
+            failed handler = "echo '!!!!!EVENT!!!!!' "
 \end{lstlisting}
 
+\lstset{language=suiterc}
+\begin{lstlisting}
+[runtime]
+    [[foo]]
+        retry delays = PT0S, PT30S
+        script = test "${CYLC_TASK_TRY_NUMBER}" -eq 2
+        [[[events]]]
+            handler = "echo '!!!!!EVENT!!!!!' "
+            handler events = retry, failed
+\end{lstlisting}
+
+N.B. The handler commands is equivalent to
+\lstinline="echo '!!!!!EVENT!!!!!' '%(event)s' '%(suite)s' '%(id)s' '%(message)s'"=.
```

Comment:
This line runs off the page badly.  Also, a minor point, but I think "N.B." is not widely used outside of the UK; probably "Note" or "Note that" would be clearer to others.